File: summer16/Week09/arguments.py
# arguments.py
#
#
#
# B. Bird - 06/28/2016

import sys

#The command line arguments to the program are stored in sys.argv,
#which is a list.

print("Number of arguments: %d"%len(sys.argv))

arg_num = 0
for arg in sys.argv:
	print("Argument %d: '%s'"%(arg_num, arg))
	arg_num = arg_num + 1

File: summer16/Week02/arguments.c
/* arguments.c


   B. Bird - 05/11/2016
*/

#include <stdio.h>

int main(int argc, char* argv[]){
	
	int i;
	printf("Number of command line arguments: %d\n",argc);
	for (i = 0; i < argc; i++){
		printf("Argument %d: \"%s\"\n",i,argv[i]);
	}
	
	return 0;
}
